From aec4b5228d1536452b1816731c41d9b37de2a25c Mon Sep 17 00:00:00 2001
From: Petr Uzel <petr.uzel@suse.cz>
Date: Mon, 15 Oct 2012 10:31:53 +0200
Subject: [PATCH 39/89] doc: update partprobe manpage

* doc/C/partprobe.8: Add long options, REPORTING BUGS section
and adjust DESCRIPTION section because it wasn't quite correct.
---
 doc/C/partprobe.8 | 13 +++++++------
 1 file changed, 7 insertions(+), 6 deletions(-)

diff --git a/doc/C/partprobe.8 b/doc/C/partprobe.8
index 6abf97d..48ae5dc 100644
--- a/doc/C/partprobe.8
+++ b/doc/C/partprobe.8
@@ -31,22 +31,23 @@ command.
 .\" \fI<whatever>\fP escape sequences to invode bold face and italics,
 .\" respectively.
 \fBpartprobe\fP is a program that informs the operating system kernel of
-partition table changes, by requesting that the operating system re-read
-the partition table.
+partition table changes.
 .SH OPTIONS
 This program uses short UNIX style options.
 .TP
-.B \-d
+.B -d, --dry-run
 Don't update the kernel.
 .TP
-.B \-s
+.B -s, --summary
 Show a summary of devices and their partitions.
 .TP
-.B \-h
+.B -h, --help
 Show summary of options.
 .TP
-.B \-v
+.B -v, --version
 Show version of program.
+.SH REPORTING BUGS
+Report bugs to <bug-parted@gnu.org>
 .SH SEE ALSO
 .BR parted (8).
 .SH AUTHOR
-- 
1.8.5.3

